### ISSUE TITLE
loss=NAN  什么原因

### ISSUE THREAD

**Issue by ghost** [shawnspace/HRAN#10]
程序运行的时候，输出如下

C:\Users\...\site-packages\h5py\__init__.py:36: FutureWarning: Conversion of the second argument of issubdtype from `float` to `np.floating` is deprecated. In future, it will be treated as `np.float64 == np.dtype(float).type`.
  from ._conv import register_converters as _register_converters
INFO:tensorflow:Found 5054 out of 40477 vectors in ./data/glove.txt
INFO:tensorflow:model dir ./model/persona_chat1
INFO:tensorflow:check point None
INFO:tensorflow:Begin training
INFO:tensorflow:global step 100 loss: nan
INFO:tensorflow:global step 200 loss: nan

**Comment by ghost** [comment]
能否请大神解答一下 loss=nan 可能的原因是什么？

目前的参数

tf.flags.DEFINE_string('word_embed_path','./data/glove.txt','path to word embedding')
tf.flags.DEFINE_string('vocab_path','./data/rg_vocab.txt','vocab path')
tf.flags.DEFINE_integer('vocab_size',18423,'vocab size')
tf.flags.DEFINE_integer("word_dim", 300, "Dimensionality of the embeddings")
tf.flags.DEFINE_integer('word_rnn_num_units', 600, 'Num of rnn cells')
tf.flags.DEFINE_integer('context_rnn_num_units', 1200, 'Num of rnn cells')
tf.flags.DEFINE_integer('decoder_rnn_num_units', 1200, 'Num of rnn cells')
tf.flags.DEFINE_integer('context_attn_units',100,'num context attn units')
tf.flags.DEFINE_integer('utte_attn_units',100,'num utterance level attn units')
tf.flags.DEFINE_integer('beam_width', 10, 'Num of beam_width')
tf.flags.DEFINE_float('keep_prob', 1.0, 'the keep prob of rnn state')
tf.flags.DEFINE_string('rnn_cell_type', 'GRU', 'the cell type in rnn')

# Pre-trained parameters
tf.flags.DEFINE_integer('max_sentence_length', 25,'the max sentence length')
tf.flags.DEFINE_integer('max_context_length', 20,'the max context length')

# Training Parameters,
# train example 131438 4000 step /epoch
# valid example 3907
# test example 3894
tf.flags.DEFINE_integer("batch_size", 32, "Batch size during training")
tf.flags.DEFINE_integer("eval_batch_size", 64, "Batch size during evaluation")
tf.flags.DEFINE_integer('num_epochs', 10, 'the number of epochs')
tf.flags.DEFINE_integer('eval_step', 2000, 'eval every n steps')
tf.flags.DEFINE_boolean('shuffle_batch',True, 'whether shuffle the train examples when batch')
tf.flags.DEFINE_float("learning_rate", 0.0001, "Learning rate")  #07112A
tf.flags.DEFINE_integer('summary_save_steps',200,'steps to save summary')
